src/Cache/_ScsDataClient.php

```diff
@@ -271,18 +281,19 @@ public function listFetch(string $cacheName, string $listName): CacheListFetchRe
     }
 
     public function listPushFront(
-        string $cacheName, string $listName, string $value, bool $refreshTtl, ?int $truncateBackToSize = null, ?int $ttlSeconds = null
+        string $cacheName, string $listName, string $value, ?int $truncateBackToSize = null, CollectionTtl $ttl = null
```

Comment:
Same as above: please make all the CollectionTtl parameters in this file explicitly nullable `?CollectionTtl` instead.